site2/website/release-notes.md

```diff
@@ -1,6 +1,191 @@
+* Fixed the argument and description for dead letter topic is wrong [#6101](https://github.com/apache/pulsar/pull/6101)  
+* [broker] Fixed bug that backlog message that has not yet expired could be deleted due to TTL [#6211](https://github.com/apache/pulsar/pull/6211)  
+* [Broker] Remove problematic semicolon from conf [#6303](https://github.com/apache/pulsar/pull/6303)  
+* [Zookeeper] Fixed casting in ZooKeeperCache.getDataIfPresent() [#6313](https://github.com/apache/pulsar/pull/6313)  
+* [Broker] Fixed broker to specify a list of bookie groups [#6349](https://github.com/apache/pulsar/pull/6349)  
+* Fixed Amend the default value of `supportedNamespaceBundleSplitAlgorithms` [#6374](https://github.com/apache/pulsar/pull/6374)  
+* Fix maven broken link [#6068](https://github.com/apache/pulsar/pull/6068)  
+* [Minor] Fixed java code errors reported by lgtm. [#6398](https://github.com/apache/pulsar/pull/6398)  
+* Consumer received duplicated deplayed messages upon restart [#6404](https://github.com/apache/pulsar/pull/6404)  
+* Fixed the max backoff configuration for lookups [#6444](https://github.com/apache/pulsar/pull/6444)  
+* Pin the netty-transport-native-epoll to avoid conflicts [#6194](https://github.com/apache/pulsar/pull/6194)  
+* Fix java doc for key shared policy. [#6341](https://github.com/apache/pulsar/pull/6341)  
+* Seek to the first one >= timestamp [#6393](https://github.com/apache/pulsar/pull/6393)  
+
+
+#### Enhancements
+* [Client] Make tests more stable by using JSONAssert equals [#6247](https://github.com/apache/pulsar/pull/6247)  
+* [Broker] Enable get precise backlog and backlog without delayed messages. [#6310](https://github.com/apache/pulsar/pull/6310)  
+* [ZooKeeper] Upgrade ZooKeeper to 3.5.7 [#6329](https://github.com/apache/pulsar/pull/6329)  
+* [Managed Ledger] Avoid using same OpAddEntry between different ledger handles [#5942](https://github.com/apache/pulsar/pull/5942)  
+* [Broker] Clean up closed producer to avoid publish-time for producer [#5988](https://github.com/apache/pulsar/pull/5988)  
+* [Client] make acker in BatchMessageIdImpl transient [#6064](https://github.com/apache/pulsar/pull/6064)  
+* [Broker] Support delete inactive topic when subscriptions caught up [#6077](https://github.com/apache/pulsar/pull/6077)  
+* [Broker] Add a message on how to make log refresh immediately when starting a component [#6078](https://github.com/apache/pulsar/pull/6078)  
+* [Pulsar Admin] allow tenant admin to manage subscription permission [#6122](https://github.com/apache/pulsar/pull/6122)  
+* [Broker] Output resource usage rate to log on broker [#6152](https://github.com/apache/pulsar/pull/6152)  
+* [Broker] Introduce maxMessagePublishBufferSizeInMB configuration to avoid broker OOM [#6178](https://github.com/apache/pulsar/pull/6178)  
+* Create managed ledger path on local zookeeper when create partitions [#6189](https://github.com/apache/pulsar/pull/6189)  
```

Comment:
This one is fixes problems that #5572 introduced, so we don't need to contain this one in the release notes.